GVFS/GVFS.Virtualization/FileSystemCallbacks.cs

```diff
@@ -793,7 +826,7 @@ private FileSystemTaskResult AddModifiedPathAndRemoveFromPlaceholderList(string
 
         private FileSystemTaskResult PostBackgroundOperation()
         {
-            this.modifiedPaths.ForceFlush();
+            this.modifiedPaths.WriteAllEntriesAndFlush();

```

Comment:
Why did this need to change, and what is the difference from before?

Reply:
This was changed so that the ModifiedPaths file will be clean after the background operations, meaning that there will only be the "A path" entries.  Otherwise the file will become littered with "A path", "D path" entries and would never get cleaned up.